apps/meteor/client/components/message/toolbox/MessageActionMenu.tsx

```diff
@@ -26,6 +26,7 @@ const MessageActionMenu = ({ options, ...props }: MessageActionMenuProps): React
 		.map(({ color, ...option }) => ({
 			...option,
 			...(color === 'alert' && { variant: 'danger' as const }),
+			...(option.id === 'reply-directly' && { role: 'link' as const }),
```

Comment:
Please, pass the role here `apps/meteor/app/ui-utils/client/lib/messageActionDefault.ts` Line: 28

Reply:
Done! Thanks!